.github/workflows/linux.yml

```diff
@@ -398,9 +398,141 @@ jobs:
           ${{ env.INSTALL_DIR }}/samples_bin/whisper_speech_recognition ./whisper-tiny/ how_are_you_doing_today.wav
         working-directory: ${{ env.MODELS_DIR }}
 
+  genai_build_nodejs_bindings:
+    name: Build Node.js bindings
+    strategy:
+      fail-fast: false
+      matrix:
+        build-type: [Release]
+    needs: [ openvino_download ]
+    timeout-minutes: 20
+    defaults:
+      run:
+        shell: bash
+    runs-on: aks-linux-4-cores-16gb
+    container:
+      image: openvinogithubactions.azurecr.io/ov_build/ubuntu_22_04_x64:${{ needs.openvino_download.outputs.docker_tag }}
+      volumes:
+        - /mount:/mount
+      options: -e SCCACHE_AZURE_BLOB_CONTAINER -e SCCACHE_AZURE_CONNECTION_STRING -v ${{ github.workspace }}:${{ github.workspace }}
+    env:
+      CMAKE_GENERATOR: Unix Makefiles
+      OV_INSTALL_DIR: ${{ github.workspace }}/ov
+      INSTALL_DIR: ${{ github.workspace }}/install
+      BUILD_DIR: ${{ github.workspace }}/build
+      SRC_DIR: ${{ github.workspace }}/src
+
+    steps:
+      - name: Clone openvino.genai
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          path: ${{ env.SRC_DIR }}
+          submodules: recursive
+
+      - name: Download OpenVINO package
+        uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 # v4.1.8
+        with:
+          name: ${{ needs.openvino_download.outputs.ov_artifact_name }}
+          path: ${{ env.OV_INSTALL_DIR }}
+          merge-multiple: true
+
+      - name: Build with ENABLE_JS=ON
+        run: |
+          source ${{ env.OV_INSTALL_DIR }}/setupvars.sh
+          cmake -DCMAKE_BUILD_TYPE=${{ matrix.build-type }} -DENABLE_JS=ON -S ${{ env.SRC_DIR}} -B ${{ env.BUILD_DIR }}
+          cmake --build ${{ env.BUILD_DIR}} --config ${{ matrix.build-type }} --parallel $(nproc) --verbose
+          cmake --install ${{ env.BUILD_DIR }} --config ${{ matrix.build-type }} --prefix ${{ env.INSTALL_DIR }}
+
+      - name: Combine binaries for Node.js package
+        run: |
+          mkdir -p nodejs
+          cp runtime/lib/intel64/* nodejs
+          cp runtime/3rdparty/tbb/lib/* nodejs
+          cp genai_node_addon.node nodejs
+          patchelf --set-rpath '$ORIGIN' libopenvino.so.2025.0.0
```

Comment:
OpenVINO provider returns output called `ov_version` . 
Or maybe take the version from other places (parse some GenAI files, e.g. project.toml)

I suppose you can parse it and use the version instead of hardcoded 2025.0.0

Reply:
do we need to patch other libs? I see more places

![{173B9D17-02E4-403A-952E-EE407AE56361}](https://github.com/user-attachments/assets/16b324f5-82dd-4772-995d-9ad1079d6e65)

Reply:
Can we move this logic into cmake project?

Reply:
no, because it patches prebuilt OpenVINO which is not GenAI responsibility